### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Source tab_Screen Reader: When the focus is on "Details" menu button, Narrator is not reading anything

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4301]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ScreenReader; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev1; #Narrator 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator

**MAS Violated:** MAS 1.3.1

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application 
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Select "Source" tab 
8.Tab till "Details" button and check if the Narrator is reading about it

**Actual Result:**
When the focus is on "Details" button, Narrator is not reading anything

**NOTE:** NVDA is reading it properly

**Expected Result:**
When the focus is on "Details" button, Narrator should read about it

**User impact:**
Screen Reader users will not be able to access the details under "Details" button

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true)

[Bug_4301_Java Tooling_source tab_Details_Narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/4469368/Bug_4301_Java.Tooling_source.tab_Details_Narrator.pptx)

![Bug_4301_Java Tooling_Source tab_Details_NVDA_No issue](https://user-images.githubusercontent.com/41108985/79118393-73914000-7dab-11ea-9656-897e5160df35.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: minimal, somewhat important, important, crucial, 
+ Quality of experience: annoying. Narrator could not read "details" button but the collapse\expand button works and the information of details display, user is not blocked 
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by soumyatippapuram** [comment]
@jingyanjingyan could you please provide the tracking bug link here

**Comment by jingyanjingyan** [comment]
@soumyatippapuram I have added the tracking bug link in the previous comment, please help review it. BTW,  in the summary part, you mentioned about
> User impact:
Screen Reader users will not be able to access the details under "Details" button

But in fact, It does not block the create project process, and user could access the details infomation. Do you agree with this?

**Comment by soumyatippapuram** [comment]
@jingyanjingyan Does the "Details" button, by default, be in expanded state? If yes, it doesn't block the user. If no, it does blocks the user.

**Comment by jingyanjingyan** [comment]
@soumyatippapuram The "Details" button default in expanded state.

**Comment by soumyatippapuram** [comment]
@jingyanjingyan then that is non-blocking. I will get it reviewed by our SME and take action on this issue

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Eclipse wizard. You can find the same dialog in New Project > Java > Java Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse without our toolkit.
![image](https://user-images.githubusercontent.com/36756024/81391405-9830cb80-914f-11ea-8f3b-949ef44970c9.png)

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749651
